### PR TITLE
remediation: AWS ELBv2 No Instances remediated, added test cases

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

**Review by dipsubha06 — COMMENTED** [review]
Is it a good idea to delete the resource entirely? since these are customer resource. will remediation be required for this plugin.

**Review by dipsubha06 — COMMENTED** [review]
Is it a needed to be delete the resources since they belong to user. is this plugin needs remediation?

**Comment by AkhtarAmir** [comment]
Yes so me Gio and I discussed about this and we decided this to be manual remediation which means it won't get triggered automatically. User will have to manually run this so I think its fine

**Review by dipsubha06 — APPROVED** [review]

**Review by giorod3 — APPROVED** [review]

File: plugins/aws/elbv2/elbv2NoInstances.js [hunk 1]
```diff
@@ -53,5 +65,43 @@ module.exports = {
...
+        config.region = lbNameArr[3];
+
+        // create the params necessary for the remediation
+        var params = {
+            'LoadBalancerArn': resource
+        };
+
+        var remediation_file = settings.remediation_file;
+        remediation_file['pre_remediate']['actions'][pluginName][resource] = {
+            'Deletion': 'NotDeleted',
```
Comment (giorod3) [inline plugins/aws/elbv2/elbv2NoInstances.js:83]:
```suggestion
            'Deletion': 'NOT_DELETED,
```

**Review by giorod3 — APPROVED** [review]

File: plugins/aws/elbv2/elbv2NoInstances.js [hunk 2]
```diff
@@ -53,5 +65,42 @@ module.exports = {
...
+        config.region = lbNameArr[3];
+
+        // create the params necessary for the remediation
+        var params = {
+            'LoadBalancerArn': resource
+        };
+
+        var remediation_file = settings.remediation_file;
+        remediation_file['pre_remediate']['actions'][pluginName][resource] = {
+            'Deletion': 'NOT_DELETED,
```
Comment (giorod3) [inline plugins/aws/elbv2/elbv2NoInstances.js:83]:
```suggestion
            'Deletion': 'NOT_DELETED',
```

**Review by giorod3 — APPROVED** [review]

**Pull request merged**